pkg_api/server/__init__.py

```diff
@@ -49,6 +54,35 @@ def create_app(testing: bool = False) -> Flask:
     api.add_resource(ServiceManagementResource, "/service")
     api.add_resource(PersonalFactsResource, "/facts")
     api.add_resource(PKGExplorationResource, "/explore")
-    api.add_resource(NLResource, "/nl")
+    api.add_resource(
+        NLResource,
+        "/nl",
+        resource_class_kwargs={"nl_to_pkg": _init_nl_to_pkg(app.config)},
+    )
 
     return app
+
+
+def _init_nl_to_pkg(config: Config) -> NLtoPKG:
+    """Initializes the NL to PKG module.
+
+    Args:
+        config: BaseConfig object.
```

Comment:
Nit: Maybe 'Server configuration' will be more descriptive.

Reply:
Done.